scripts/clinical-clusters/ema-clinical-cluster-builder.js

```diff
@@ -25,53 +25,129 @@ const MAX_SPECIALITY_SET_SIZE = 6;
+      {measureId: '112', optionals: ['113']},
+      {measureId: '113', optionals: ['112']}
+    ],
+    registry: [
+      {measureId: '047', optionals: []},
+      {measureId: '110', optionals: []},
+      {measureId: '130', optionals: []},
+      {measureId: '134', optionals: []},
+      {measureId: '226', optionals: []},
+      {measureId: '317', optionals: []},
+      {measureId: '424', optionals: []},
+      {measureId: '430', optionals: []},
+      {measureId: '051', optionals: ['052']},
+      {measureId: '052', optionals: ['051']},
+      {measureId: '398', optionals: ['444']},
+      {measureId: '444', optionals: ['398']},
+      {measureId: '024', optionals: ['418']},
+      {measureId: '418', optionals: ['024']},
+      {measureId: '005', optionals: ['008']},
+      {measureId: '006', optionals: ['118', '007']},
+      {measureId: '007', optionals: ['118', '006']},
+      {measureId: '008', optionals: ['005']},
+      {measureId: '118', optionals: ['007', '006']},
+      {measureId: '426', optionals: ['427']},
+      {measureId: '427', optionals: ['426']},
+      {measureId: '112', optionals: ['113']},
+      {measureId: '113', optionals: ['112']}
+    ]
+  },
+  2018: {
+    claims: [
+      {measureId: '130', optionals: []},
+      {measureId: '317', optionals: []},
+      {measureId: '112', optionals: ['113']},
+      {measureId: '113', optionals: ['112']}
+    ],
+    registry: [
+      {measureId: '110', optionals: []},
+      {measureId: '130', optionals: []},
+      {measureId: '226', optionals: []},
+      {measureId: '424', optionals: []},
+      {measureId: '430', optionals: []},
+      {measureId: '051', optionals: ['052']},
+      {measureId: '052', optionals: ['051']},
+      {measureId: '398', optionals: ['444']},
+      {measureId: '444', optionals: ['398']},
+      {measureId: '024', optionals: ['418']},
+      {measureId: '418', optionals: ['024']},
+      {measureId: '006', optionals: ['118', '007']},
+      {measureId: '007', optionals: ['118', '006']},
+      {measureId: '118', optionals: ['007', '006']},
+      {measureId: '005', optionals: ['008']},
+      {measureId: '008', optionals: ['005']},
+      {measureId: '426', optionals: ['427']},
+      {measureId: '427', optionals: ['426']},
+      {measureId: '112', optionals: ['113']},
+      {measureId: '113', optionals: ['112']}
+    ]
+  }
 };
 
-function curate(clusterMap, relations) {
-  // remove clincalClusters from measures that belongs to multiple cluster
+function curateSpecialtySet(clusterMap, relations) {
+  // handle removal of sets/measures
+  if (relations) {
+    relations.forEach(r => {
+      if (r.action === 'remove') {
+        r.measureIds.forEach(m => {
+          const specialtySets = clusterMap.get(m).specialtySets;
+          const specialtySetIndex = specialtySets.findIndex(ss => ss.name === r.name);
+          specialtySets.splice(specialtySetIndex, 1);
+        });
+      } else if (r.action === 'override') {
```

Comment:
If there are any useful comments for the intent of this function that'd be great to add, particularly for "override" (as a reader with no context on any of this clinical clusters code)

Reply:
Thanks for the feedback!  I'll add some comments to make my intent more obvious.